docs/DeveloperGuide.md

```diff
@@ -426,6 +438,95 @@ In the diagram, the predicates `modulePredicate` and `taskPredicate` are the cus
 
 <img src="images/CdSequenceDiagram.png" width="1000" />
 
+### Marking modules as done
+
+Section by : [Marciano](https://github.com/midnightfeverrr)
+
+Marking modules as done allows users to selectively hide modules that they have completed.
```

Comment:
abit ambiguous, marking as done does not 'hide' it on the interface, perhaps could be refactored to something `Allows users to mark their modules as done, and alternate between viewing all modules and viewing undone modules via the list command`